arm_robots/src/arm_robots/med.py

```diff
@@ -0,0 +1,219 @@
+
+        left_arm_command = MotionCommand()
+        left_arm_command.header.frame_id = 'med_base'
+        left_arm_command.cartesian_pose = pose_stamped.pose
+        left_arm_command.control_mode = self.get_control_mode()
+        while self.arm_command_pub.get_num_connections() < 1:
+            rospy.sleep(0.01)
+
+        self.arm_command_pub.publish(left_arm_command)
+
+    # TODO: send_delta_cartesian_command
+
+    def get_joint_positions_map(self) -> Dict[str, float]:
+        arm_joint_vals = jvq_to_list(self.arm_status_listener.get().measured_joint_position)
+        return {n: val for n, val in zip(ARM_JOINT_NAMES, arm_joint_vals)}
+
+    def get_joint_positions(self, joint_names: Sequence[str] = ARM_JOINT_NAMES):
+        """
+        :args joint_names an optional list of names if you want to have a specific order or a subset
+        """
+        position_of_joint = self.get_joint_positions_map()
+
+        print(position_of_joint)
+        return [position_of_joint[name] for name in joint_names]
+
+
+class Med(MoveitEnabledRobot, BaseMed):
+
+    def __init__(self, robot_namespace: str = 'med', force_trigger: float = -0.0, **kwargs):
+        MoveitEnabledRobot.__init__(self,
+                                    robot_namespace=robot_namespace,
+                                    arms_controller_name='arm_trajectory_controller',
+                                    force_trigger=force_trigger, **kwargs)
+        BaseMed.__init__(self, robot_namespace=robot_namespace)
+        self.arm_group = 'kuka_arm'
+        self.wrist = 'med_kuka_link_ee'
+
+    def get_arm_joints(self):
+        return ARM_JOINT_NAMES
+
+    def set_control_mode(self, control_mode: ControlMode, vel=0.1, **kwargs):
+        super().set_control_mode(control_mode, vel, **kwargs)
+        self._max_velocity_scale_factor = vel
+
+    def set_grasping_force(self, force):
+        if type(force) != float or force > 80.0 or force <= 0.0:
+            print("Bad grasp force value provided! Not setting grasping force.")
+            return None
+
+        rospy.wait_for_service('/wsg_50_driver/set_force')
+        try:
+            force_proxy = rospy.ServiceProxy('/wsg_50_driver/set_force', Conf)
+            force_resp = force_proxy(force)
+            return force_resp.error
+        except rospy.ServiceException as e:
+            print("Service call failed: %s" % e)
+
+    def grasp(self, width, speed=50.0):
+        rospy.wait_for_service('/wsg_50_driver/grasp')
+        try:
+            close_proxy = rospy.ServiceProxy('/wsg_50_driver/grasp', Move)
+            grasp_resp = close_proxy(width, speed)
+            return grasp_resp.error
+        except rospy.ServiceException as e:
+            print("Service call failed: %s" % e)
+
+    def open_gripper(self):
+        rospy.wait_for_service('/wsg_50_driver/move')
+        try:
+            move_proxy = rospy.ServiceProxy('/wsg_50_driver/move', Move)
```

Comment:
be careful with the leading `/` here, what if you have to med arms and want to talk to one versus the other?